specification/maps/data-plane/Weather/preview/1.0/examples/GetStormForecast.json

```diff
@@ -0,0 +1,9787 @@
+{
```

Comment:
I can't find the example API call at https://review.docs.microsoft.com/en-us/rest/api/documentation-preview/weather/storm-forecast in this view that is at, but that example doesn’t work. I get a "govId is missing or empty" error. Is this because it’s difficult to have working example for a forecast as a tropical storm may not exist at the moment?  i.e.: https://atlas.microsoft.com/weather/tropical/storms/forecasts/json?api-version=1.0&year=2021&basinId=NP&governmentId=2 However, AccuWeather’s docs have a working example of https://api.accuweather.com/tropical/v1/gov/storms/2018/NP/26/forecasts?apikey=YourKey. I tried that example using our API and it doesn’t work for some reason?

Reply:
As our offline discussion, it should works now.
Please let me know if the problem happens again.